### DISCUSSION TITLE
New voices and voice list

### DISCUSSION THREAD

**Discussion by St33lMouse** [snakers4/silero-models#219] [Q&A]
Where do you find the list of voices? Is it possible to make new voices?

**Comment by snakers4** [comment]
> Where do you find the list of voices? 

https://github.com/snakers4/silero-models/blob/a7e61e1c6d69e981d2b8da39194ed7512ad4c3c2/models.yml#L144

> Is it possible to make new voices?

Yes, commercially

**Reply by luthis1124** [reply]
How much does that cost? Do you have a link?

**Comment by St33lMouse** [comment]
Ah, I figured. That's too bad.
